cli/util/progress_bar/renderer.rs

```diff
@@ -116,40 +120,82 @@ impl ProgressBarRenderer for BarProgressBarRenderer {
 }
 
 #[derive(Debug)]
-pub struct TextOnlyProgressBarRenderer;
+pub struct TextOnlyProgressBarRenderer {
+  last_tick: AtomicUsize,
+  start_time: std::time::Instant,
+}
+
+impl Default for TextOnlyProgressBarRenderer {
+  fn default() -> Self {
+    Self {
+      last_tick: Default::default(),
+      start_time: std::time::Instant::now(),
+    }
+  }
+}
 
 impl ProgressBarRenderer for TextOnlyProgressBarRenderer {
   fn render(&self, data: ProgressData) -> String {
-    let bytes_text = {
-      let total_size = data.display_entry.total_size;
-      let pos = data.display_entry.position;
-      if total_size == 0 {
-        String::new()
-      } else {
-        format!(
-          " {}/{}",
-          human_download_size(pos, total_size),
-          human_download_size(total_size, total_size)
-        )
-      }
+    let last_tick = {
+      let last_tick = self.last_tick.load(Ordering::Relaxed);
+      let last_tick = (last_tick + 1) % 8;
+      self.last_tick.store(last_tick, Ordering::Relaxed);
+      last_tick
     };
+    let current_time = std::time::Instant::now();
+
+    const SPINNER_CHARS: [&'static str; 8] =
+      ["⣷", "⣯", "⣟", "⡿", "⢿", "⣻", "⣽", "⣾"];
```

Comment:
I've always prefered "/", "-", "\", "|" - wdyt?

Reply:
No strong preferences:

https://github.com/user-attachments/assets/0dd6f88e-f8ba-46e0-9908-c1c667312231



https://github.com/user-attachments/assets/d22394fc-ba24-45a6-912b-9af102ed1e41

Reply:
Look old school. Maybe we should do something do something more distinctive 
```
["✦", "★", "✶", "✷", "✸", "✹", "✺"]
```

Reply:
Ryan's suggestion: ![CleanShot 2024-08-06 at 17 48 06](https://github.com/user-attachments/assets/9cf9ae42-04f3-47f8-9730-ba917ff566f5)

I still prefer the current one.

Reply:
Other options

```
["🌕", "🌖", "🌗", "🌘", "🌑", "🌒", "🌓", "🌔"]
["[   ]", "[=  ]", "[== ]", "[===]", "[ ==]", "[  =]"]
```

Reply:
![CleanShot 2024-08-07 at 10 35 14](https://github.com/user-attachments/assets/1f3d4f8c-9f1f-4fb7-97c3-c91cb446dce8)
![CleanShot 2024-08-07 at 10 37 24](https://github.com/user-attachments/assets/5b06a352-eff6-468f-b1c8-4f957f055687)
![CleanShot 2024-08-07 at 10 38 27](https://github.com/user-attachments/assets/df51333b-5304-481e-8af6-ea70ac3d2b12)

Reply:
I still think the original one (8 dots) is the best, followed by the last one (3 horizontal dots).

Which one do we want to go with?

Reply:
I like the original one the most too. That said anything that shows that it's doing something is fine to me as well.

Reply:
Original one then